libs/content-search-toolkit/src/queries/search.ts

```diff
@@ -102,14 +109,29 @@ export const searchQuery = (
 
   return {
     query: {
-      bool: {
-        should,
-        must,
-        must_not: mustNot,
-        minimum_should_match: minimumShouldMatch,
+      function_score: {
+        query: {
+          bool: {
+            should,
+            must,
+            must_not: mustNot,
+            minimum_should_match: minimumShouldMatch,
+          },
+        },
+        functions: [
+          {
+            field_value_factor: {
+              field: 'popularityScore',
+              factor: 1.2,
+              modifier: 'log1p',
+              missing: 1,
+            },
+          },
+          { filter: { range: { processEntryCount: { gte: 1 } } }, weight: 2 },
+        ],
       },
     },
-    ...(Object.keys(aggregation.aggs).length ? aggregation : {}), // spread aggregations if we have any
+    ...(Object.keys(aggregation.aggs).length ? aggregation : {}), // spread aggregations if we have any,
```

Comment:
Perhaps the comma in the comment is unintentional :P

Reply:
well spotted ... fixed